### ISSUE TITLE
Android: New request to authority not possible

### ISSUE THREAD

**Issue by Ahoybrause** [jfilter/frag-den-staat-app#114]
A request to an authority is not available in Android, because no authority can be selected after entering it into the search field.

**Comment by jfilter** [comment]
It looks like @stefanw changed the API. It will stay broken since I'm not working on the app anymore. I'm considering pulling the app from the play / app store. :/

**Comment by stefanw** [comment]
Ha, sorry! But maybe for the best.

**Comment by stefanw** [comment]
@jfilter If you have some details, I can also see if it can be fixed.

**Comment by jfilter** [comment]
The endpoint for search for public bodies must have changed: https://fragdenstaat.de/api/v1/publicbody/search/

Is this enough information? I'm not sure if I get the app to run again. 😬

**Comment by stefanw** [comment]
Yes, bad commit here: https://github.com/okfde/froide/commit/fc434cf7e763f08c249d5fb68549ef0e1fa3d8f3
It even says it's backward incompatible. 🥲 

I should have applied a version bump I guess. Not sure if it's worth it.

**Comment by jfilter** [comment]
I'll leave the app in the stores, but it's unlikely that I will provide a new version with a fix.
